src/content/mod.rs

```diff
@@ -51,6 +66,31 @@ impl File for Content
     }
 }
 
+impl Observed for Content
+{
+    fn stale(&self) -> bool
+    {
+        match modified(&self.disk_path)
+        {
+            Some(t) =>
+            {
+                if t <= self.last_refreshed
+                {
+                    return false;
+                }
+            },
+            None => ()
+        }
+
+        return file_hash(&self.disk_path) != self.hash
+    }
+
+    fn refresh(&mut self)
+    {
+        let _ = self.load_from_file();
+    }
+}
```

Comment:
Add a test to check these functions work cross platform, and generally

Reply:
Seems the file modified time does not work that well when calls are made in quick succession. I.e. if a file is modified right after reading the modified time may still be lower than the time stored in Content... also may not be cross platform, checking the hash is an additional 400% but can still do 200,000 checks/s